src/components/DateRangePicker.jsx

```
@@ -138,7 +138,9 @@ export default class DateRangePicker extends React.Component {
 
   onEndDateChange(endDateString) {
     const { startDate, isOutsideRange, onDatesChange, onFocusChange, displayFormat } = this.props;
-    const endDate = toMomentObject(endDateString, displayFormat);
+
+    const formatString = typeof displayFormat === 'function' ? displayFormat() : displayFormat;
```

i'm a big fan of failing fast and loud